tests/x509/test_x509.py

```diff
@@ -2646,7 +2646,9 @@ def test_sign_ec_with_md5(self, backend):
     def test_build_cert_with_dsa_private_key(
         self, hashalg, hashalg_oid, backend
     ):
-        if backend._fips_enabled and hashalg is hashes.SHA1:
+        if (
+            backend._fips_enabled and hashalg is hashes.SHA1
+        ):  # pragma: no cover
```

Comment:
If we always skip FIPS due to the dsa_supported check returning False when FIPS is enabled, this entire branch can be removed, right?